src/servers/middlewares/auth-router.ts

```diff
@@ -114,4 +121,135 @@ authRouter.post(
   }),
 )
 
+authRouter.post(
+  "/login",
+  wrapAsyncToRunInSpan({
+    namespace: "auth-router",
+    fn: async (req: express.Request, res: express.Response) => {
+      try {
+        const ipString = isDev ? req?.ip : req?.headers["x-real-ip"]
+        const ip = parseIps(ipString)
+        if (ip === undefined) {
+          throw new Error("IP is not defined")
+        }
+
+        const phone = req.body.phoneNumber as PhoneNumber
```

Comment:
should verify type with `checkedToPhoneNumber`. this is typically done in the use case not in the network layer.